src/dbcp/etl.py

```diff
@@ -186,7 +186,16 @@ def etl_gridstatus_isoqueues():
 
 def etl_manual_ordinances() -> dict[str, pd.DataFrame]:
     """ETL manually maintained ordinances."""
-    raw_dfs = dbcp.extract.manual_ordinances.extract()
+    county_level_control_uri = (
+        "airtable/Clean Build State Permitting Analysis/County Level Control.json"
+    )
+
+    es = ExtractionSettings.from_yaml("/app/dbcp/settings.yaml")
+    es.update_archive_generation_numbers()
```

Comment:
For my understanding: this function does:

* if there wasn't a generation specified in settings.yaml, finds the latest generation num so everything downstream of this happens with a concrete generation number
* otherwise, grab the pinned one

Is that right?

If so, we might also consider calling this `update_archive_generation_numbers` *in* `from_yaml` so that we never forget to do it.